src/controllers/LiveDataController.ts

```diff
@@ -65,10 +73,13 @@ async function updateProcessMemoryMetrics(payload: string | LocalLiveProcess) {
 
     const heapMemMetrics = await getMemoryMetrics(processKey, "heapMemory");
     const nonHeapMemMetrics = await getMemoryMetrics(processKey, "nonHeapMemory");
-    await vscode.commands.executeCommand("setContext", "spring.memoryGraphs:hasLiveProcess", liveProcess !== undefined);
-    memoryProvider.refreshLiveHeapMemoryMetrics(liveProcess, heapMemMetrics);
-    memoryProvider.refreshLiveNonHeapMemoryMetrics(liveProcess, nonHeapMemMetrics);
-    store.data.set(processKey, { heapMemMetrics, nonHeapMemMetrics});
+
+    if(heapMemMetrics || nonHeapMemMetrics) {
+        await vscode.commands.executeCommand("setContext", "spring.memoryGraphs:hasLiveProcess", liveProcess !== undefined);
```

Comment:
I think this check is what I expect. Previously it just shows the memory view without plotting any data points.